src/screencast/wlr_screencast.c

```diff
@@ -320,28 +340,39 @@ static void wlr_registry_handle_add(void *data, struct wl_registry *reg,
 		uint32_t id, const char *interface, uint32_t ver) {
 	struct xdpw_screencast_context *ctx = data;
 
+	uint32_t version = 0;
```

Comment:
I like the idea of getting rid of these magic numbers, but I think I'd rather declare additional constants (like SC_MANAGER_VERSION) for each of these interfaces, and if you need to do some comparing/checking like you're doing on L#358, just keep the variables scoped to the if block.

Reply:
Ok. Good idea to change those to macros. Changed them.